rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -68,12 +144,39 @@ components:
     Accepted:
       description: The transformation request was accepted for processing.
       headers:
-        'Content-Version':
+        Content-Version:
           $ref: '#/components/headers/ContentVersion'
+        Location:
+          $ref: '#/components/headers/Location'
+    MetadataInformationResponse:
+      description: Information on each of the metadata types that came out of the transformation.
+      headers:
+        Content-Version:
+          $ref: '#/components/headers/ContentVersion'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/MetadataInformationResponse'
+    TransformationDeletedResponse:
+      description: The transformation request was successfully deleted.
+      headers:
+        Content-Version:
+          $ref: '#/components/headers/ContentVersion'
+    MetadataContentResponse:
+      description: Successfully retrieved.
+      headers:
```

Comment:
❗️ Might want to include `Content-Type` and `Content-Length` as well.

Reply:
`Content-Type` header will already be handled by the `application/octet-stream` mimetype specified below. I can add the `Content-Length`.